src/core/index.js

```diff
@@ -563,8 +575,9 @@ export function overrideOptions(m, opts) {
   }
 
   if (opts.flashMessage) {
-    const key = 'success' === opts.flashMessage.type ? 'globalSuccess' : 'globalError';
-    m = tset(m, key, opts.flashMessage.text);
+    const { type } = opts.flashMessage;
```

Comment:
this will throw if flashMessage.type is undefined. use `error` as a default (+ a test)

Reply:
Makes sense.

However, I just had a look at the tests and there is one test case explicitly asserting that Lock should fail if there is no `type` set on a `flashMessage`:
https://github.com/auth0/lock/blob/master/test/show_with_flash_message.test.js#L19
This kind of contradicts your suggestion to make the `type` optional and falling back to `error` as default ` type`.

I'm fine with both options, what do you prefer?

Reply:
You're right. I didn't remember that 😝 
Let's leave it like this. I'll test the PR today and merge it. Thanks! 🎉

Reply:
@luisrudge Hi Luis, did you manage to take a look at this again in the meanwhile?